config/_shared.json

```diff
@@ -36,7 +36,7 @@
 	"facebook_app_id": "611241942420191",
 	"rebrand_cities_prefix": "rebrandcitiestestsite",
     "livechat_support_locales": [ "en", "es", "pt-br" ],
-    "support_site_locales": [ "en", "es", "de", "ja", "pt-br" ],
+    "support_site_locales": [ "en", "es", "de", "fr", "he", "id", "it", "ja", "nl", "pt-br", "ru", "tr" ],
```

Comment:
We've now launched all Mag16. Could you add those and I think then we can launch this!